### ISSUE TITLE
Oxygen V4.0 Beta 2 : Alt Text Field Missing

### ISSUE THREAD

**Issue by jam1970** [soflyy/oxygen-bugs-and-features#2852]
When adding an image in Oxygen, there is no alt text field. 

1. Create an Oxygen template
2. Add an image 
3. Look for the alt text field on the primary tab - there isn't one.

![alt](https://user-images.githubusercontent.com/80896622/166046366-4b861795-daeb-4970-87bc-37c03510f69d.jpg)

**Comment by Spellhammer** [comment]
In Oxygen 4.0, Image elements default to the Media Library option. Media Library images pull their alt text from the WordPress media library.

If you want to set it explicitly, it will be possible to add an alt attribute under Advanced > Attributes and either set the value you want for that instance of the image, or pull the value dynamically.
